checker/src/main/java/org/checkerframework/checker/resourceleak/ResourceLeakAnnotatedTypeFactory.java

```diff
@@ -80,6 +97,84 @@ public ResourceLeakAnnotatedTypeFactory(final BaseTypeChecker checker) {
     this.postInit();
   }
 
+  /**
+   * Collects all the possible owning fields enclosed by the {@link ClassTree} and initializes the
+   * fieldsToFinalizers map.
```

Comment:
This method's documentation should probably also include a sentence or two describing the purpose of the `fieldsToFinalizers` map and in what contexts it makes sense to call the method.